integrations/tensorflow/bindings/python/pyiree/tf/support/module_utils.py

```diff
@@ -19,13 +19,21 @@
 import tempfile
 from typing import Any, Callable, Dict, Sequence, Set, Tuple, Type, Union
 
+from absl import flags
 from absl import logging
 import numpy as np
 from pyiree import rt
 from pyiree.tf import compiler
 from pyiree.tf.support import tf_utils
 import tensorflow.compat.v2 as tf
 
+flags.DEFINE_bool(
+    "use_crash_reproducer", True,
```

Comment:
maybe `capture_crash_reproducer` (to match LLVM's `--capture` and MLIR's `-pass-pipeline-crash-reproducer`